src/api/app/assets/javascripts/webui/user_profile.js

```diff
@@ -5,3 +5,30 @@ function moveInvolvementToContainer() { // jshint ignore:line
 
   container.prepend($('#involvement'));
 }
+
+function updateCharactersCount(e) {
+  var maxLength = $(e.target).attr('maxlength');
+  var currentLength = $(e.target).val().length;
+  var remainingCount = maxLength - currentLength;
+
+  // Change font color to "danger" when low amount of characters left
+  var style = window.getComputedStyle(document.body);
+  var defaultFontColor = style.getPropertyValue('color');
+  var dangerFontColor = style.getPropertyValue('--danger');
+  $('#bio-chars-counter').css('color', (remainingCount < 10) ? dangerFontColor : defaultFontColor);
```

Comment:
This could be replaced by [.toggleClass()](https://api.jquery.com/toggleclass/) with the `text-danger` class.

It would be like this:
```suggestion
  $('#bio-chars-counter').toggleClass('text-danger', remainingCount < 10);
```

Reply:
Done. Thank you for the suggestion!